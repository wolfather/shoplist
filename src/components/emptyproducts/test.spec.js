import EmptyProducts from './index';
import jest from 'jest';
import { render, screen } from '@testing-library/react';

describe('EmptyProducts', () => {

    test('Should have a text on a paragraph', () => {
        const textExpected = 'Sem produtos no momento';
        render(<EmptyProducts />);

        const el = screen.getBytText('0');
        expect(el.innerText).toBe(textExpected);
    });


});