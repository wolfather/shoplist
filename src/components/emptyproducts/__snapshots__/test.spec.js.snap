// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should have a snapshot 1`] = `
<p>
  Sem produtos no momento
</p>
`;
